Src/BootCamp.Chapter/Program.cs

```diff
@@ -5,8 +5,33 @@ namespace BootCamp.Chapter
     class Program
     {
         static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
+        {            
+            //Homework #1 
+            //Read name, surename, age, weight(in kg) and height(in cm) from console.
+            //1) Print all the info based on the example message below:
+            //Tom Jefferson is 19 years old, his weight is 50 kg and his height is 156.5 cm.
+            //2) Calculate and print body-mass index(BMI)
+            //3) Do 1 and 2 for another person.
```

Comment:
Person 2 missing

Reply:
Oh.  I'll add that.  I actually thought I just supposed to test with different inputs.

Reply:
I made the changes.  Do I need to do another pull request?

Reply:
I've used GitHub before but not with other people, so this is all new to me.